ui/apps/platform/src/Containers/Policies/Wizard/Step3/PolicySection.tsx

```diff
@@ -58,7 +78,13 @@ function PolicySection({ sectionIndex, descriptors, readOnly = false }: PolicySe
 
     return (
         <>
-            <Card isFlat isCompact className={!readOnly ? 'policy-section-card' : ''}>
+            <Card
+                isFlat
+                isCompact
+                isSelected={isSelected}
+                isSelectable
```

Comment:
Nit: Is it worth making this prop dependent on the feature flag as well, so that the radio button is not visible to users who have not enabled this feature?